test/apiv2/47-subnet-pools.at

```diff
@@ -0,0 +1,29 @@
+# -*- sh -*-
+#
+# subnet pools conversion tests
+#
+
+# Create a temporary containers.conf with custom DefaultSubnetPools
+cat > $TESTS_DIR/containers.default_subnet_pools.conf <<EOF
+[network]
+default_subnet_pools = [
+  { "base" = "10.89.0.0/16", "size" = 24 },
+  { "base" = "10.90.0.0/15", "size" = 26 }
+]
+EOF
+
+# Restart podman with custom config
+stop_service
+CONTAINERS_CONF=$TESTS_DIR/containers.default_subnet_pools.conf start_service
+
+# Test the Docker-compatible API endpoints
+t GET system/info 200 \
```

Comment:
looks like the endpoint is just called info

FYI you should be able to test this locally quite easily with `./test/apiv2/test-apiv2 47`